ChangeLog.md

```diff
@@ -1,6 +1,25 @@
-# wolfSSL Release x.x.x
+# wolfSSL Release 3.15.3 (6/20/2018)
 
-* Added AES performance enhancements on AMD processors with build option `./configure --enable-intelasm`
+Release 3.15.3 of wolfSSL embedded TLS has bug fixes and new features including:
+
+* ECDSA blinding added for hardening against side channel attacks
+* Fix for compatibility layer build with no server and no client defined
+* Use of optimized Intel assembly instructions on compatible AMD processor
+* wolfCrypt Nucleus port additions
+* Fix added for MatchDomainName and additional tests added
+* Fixes for building with ‘WOLFSSL_ATECC508A’ defined
+* Fix for verifying a PKCS7 file in BER format with indefinite size
+
+
+This release of wolfSSL fixes 2 security vulnerability fixes.
+
+Low level fix for PRIME + PROBE attack combined with a variant of Lucky 13. Constant time hardening was done to avoid potential cache-based side channel attacks when verifying the MAC on a TLS packet. It is recommended to update the version of wolfSSL if a potentially malicious user has access to run a parallel program on the system and inspect caching. Thanks to Eyal Ronen, Kenny Paterson, and Adi Shamir for the report.
```

Comment:
We need a recommendation here.  Who should update, which protocol versions, which type of ciphers.  For example (this is not exact) TLS 1.0 - TLS 1.2 versions using CBC cipher suites with SHA-384 need to update.